resources/com/github/aroq/drupipe/config.groovy

```diff
@@ -148,4 +148,7 @@ defaultActionParams = [
     YamlFileHandler: [
         deployFile: '.drupipe.yml',
     ],
+    Shell: [
```

Comment:
I believe we shouldn't do it for Shell, but for Config, and pass to Source from Config as action parameter.